server/src/core/symbols/function_symbol.rs

```diff
@@ -34,7 +34,7 @@ pub struct FunctionSymbol {
     pub is_static: bool,
     pub is_property: bool,
     pub doc_string: Option<String>,
-    pub ast_indexes: Vec<u16>, //list of index to reach the corresponding ast node from file ast
+    pub ast_indexes: AtomicNodeIndex,
```

Comment:
Suggestion: Maybe we could rename it to node_index? since it is just one index, not a sequence of indexes? also at symbol, class symbol. variable symbol ... ?